### DISCUSSION TITLE
Curso: 7199 | Aula: Obtendo Token do Google

### DISCUSSION THREAD

**Discussion by ddcsilva** [balta-io/forum#360] [Tira Dúvidas] [answered]
Olá Balta e comunidade,

Estou fazendo o curso 7199 - **Criando um TodoApp com ASP.NET Core, Firebase, Angular e Flutter** e estou travado na aula **Obtendo Token do Google**.

Baixei o repositório com o projeto Angular, mas não consegui executá-lo. No primeiro cenário, utilizei a versão 18.16.1 do Node e tentei instalar, mesmo sabendo que a versão do Angular no package do repositório era 9. Tentei instalar com os comandos:

- `npm install`
- `npm install --force`
- `npm install --legacy-peer-deps`

Mas não tive sucesso.

Instalei o nvm e tentei com as versões do Node: 16.10.0 e 10.13.0, juntamente com as versões do Angular CLI 8 e depois 9, também sem sucesso.

Não anotei todos os erros, mas a maioria ocorreu na hora de instalar alguma dependência. A maioria dos erros foi relacionada ao pacote do Firebase.

Não tenho muita experiência em Angular e suas compatibilidades. Pensei em fazer o curso seguinte de Web e seguir com uma versão mais atual, mas gostaria de testar este curso da API e seguir as etapas corretamente. Alguém poderia me ajudar? Obrigado!

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia, @ddcsilva, como vai?

Precisaria de algumas mensagens de erro para poder te ajudar... Realmente a instalação do Node/NVM/NPMs pode ser meio chata.
Com as mensagens conseguimos ver o que está falhando.
